internal/kafka/test/integration/kafka_create_test.go

```diff
@@ -614,7 +614,7 @@ func TestKafkaCreate_EnterpriseKafkas(t *testing.T) {
 	defer ocmServer.Close()
 
 	h, client, teardown := test.NewKafkaHelperWithHooks(t, ocmServer, func(acl *quota_management.QuotaManagementListConfig, c *config.DataplaneClusterConfig) {
-		acl.EnableInstanceLimitControl = false
```

Comment:
this was causing the various quota checks to be skipped.

For this test, we always want the quota checks to be performed.

Reply:
Good catch!